test/cmd/admin.sh

```diff
@@ -71,7 +71,8 @@ status:
 ' | oc create -f -"
 
 os::cmd::expect_success_and_text 'oadm manage-node --selector= --schedulable=true' 'Ready'
-os::cmd::expect_success_and_not_text 'oadm manage-node --selector= --schedulable=true' 'Sched'
+os::cmd::expect_success_and_not_text 'oadm manage-node --selector= --schedulable=true' 'SchedulingDisabled'
+os::cmd::expect_success_and_text 'oadm manage-node --selector= --schedulable=false' 'SchedulingDisabled'
```

Comment:
We don't have a node in `test/cmd` - that tests needs to live in e2e or extended